_episodes/15-transferring-files.md

```diff
@@ -100,18 +100,23 @@ the transfer.
 
 The most common archiving command you will use on (Linux) HPC cluster is `tar`. `tar` can be used
 to combine files into a single archive file and, optionally, compress. For example, to combine
-all files within a folder called `output_data` into an archive file called `output_data.tar` we
-would use:
+all files of the pattern `output_data/output00001.out`, `output_data/output00002.out`, ... within a folder called `output_data` into an archive file called `output_data.tar` we would use:
 
```

Comment:
Possibly too much detail here - it sounds like the command will specifically only archive files with names like `output_data/output00001.out` but the command specified will actually archive **all** files in the folder `output_data/`

Reply:
fixed with 2a3cc22